include/rive/generated/shapes/mesh_base.hpp

```diff
@@ -0,0 +1,66 @@
+#ifndef _RIVE_MESH_BASE_HPP_
+#define _RIVE_MESH_BASE_HPP_
+#include <vector>
+#include "rive/container_component.hpp"
+#include "rive/core/field_types/core_bytes_type.hpp"
+namespace rive {
+    class MeshBase : public ContainerComponent {
+    protected:
+        typedef ContainerComponent Super;
+
+    public:
+        static const uint16_t typeKey = 109;
+
+        /// Helper to quickly determine if a core object extends another without
+        /// RTTI at runtime.
+        bool isTypeOf(uint16_t typeKey) const override {
+            switch (typeKey) {
+                case MeshBase::typeKey:
+                case ContainerComponentBase::typeKey:
+                case ComponentBase::typeKey:
+                    return true;
+                default:
+                    return false;
+            }
+        }
+
+        uint16_t coreType() const override { return typeKey; }
+
+        static const uint16_t triangleIndexBytesPropertyKey = 223;
+
+    private:
+        std::vector<uint8_t> m_TriangleIndexBytes;
```

Comment:
These are the 'compressed" uint16_t values, right? i.e. compressed via uint packing?

I'm sure this is right, but I'm still confused when I see this, and its not the unpacked, 'usable' uint16_t version that we pass to the renderer. I understand we store these compressed in the file format -- I guess I just expect them to be expanded *before* we decide to store them somewhere.

Reply:
It's because Core doesn't have an understanding of the compression format. It's serialized as a bunch of bytes, so the field you currently get in the base class is a bunch of bytes. We could teach Core to be smarter about this.